src/runtime_src/core/tools/common/XBUtilities.h

```diff
@@ -87,8 +87,13 @@ namespace XBUtilities {
   std::string
   string_to_UUID(std::string str);
 
-  uint64_t 
-  string_to_bytes(std::string str);
+  enum class unit
+  {
+    bytes,
+    hertz

```

Comment:
Hertz should be capatialized for it is named after a person: Heinrich Rudolf Hertz.

Reply:
Thanks for pointing it out @rozumx , I have made changes